### ISSUE TITLE
Transfer Learning on Custom dataset / Freeze first layers and only train the top ones

### ISSUE THREAD

**Issue by anandhakrishnanh** [davidsandberg/facenet#1154]
So I have a custom dataset with only a couple of hundred classes compared to the thousands that consisted of the initial training dataset. I want to fine tune with this dataset but I don't think using the --pretrained_model in train_triplet loss will work since that would be training the entire network on the new data. Is there a way to freeze the first bottom layers since they just generalize how a face is supposed to look and only retrain the top layers which are most likely about the finner details on the face with this new dataset that I have.
